bitnami/mxnet/templates/scheduler-deployment.yaml

```diff
@@ -0,0 +1,160 @@
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{ toYaml .Values.affinity | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cloneFilesFromGit.enabled }}
+      initContainers:
+      - name: git-clone-repository
+        image: {{ include "git.image" . }}
+        imagePullPolicy: {{ .Values.git.pullPolicy | quote }}
+        command:
+        - /bin/sh
+        - -c
+        - |
+          git clone {{ .Values.cloneFilesFromGit.repository }} /app
+          cd /app
+          git checkout {{ .Values.cloneFilesFromGit.revision }}
+        volumeMounts:
+        - name: git-cloned-files
+          mountPath: /app
+      {{- end }}
+{{- if .Values.initContainers }}
+{{ toYaml .Values.initContainers | indent 6 }}
+{{- end }}
+      containers:
+      - name: mxnet
+        workingDir: {{ .Values.entrypoint.workDir }}
+        image: {{ include "mxnet.image" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: 
+        - bash
+        - -c
+        - |
+{{ include "mxnet.entrypoint" . | indent 10 }}
+        env:
+        - name: DMLC_ROLE
+          value: "scheduler"
+        - name: DMLC_NUM_WORKER
+          value: {{ .Values.workerCount | quote }}
+        - name: DMLC_NUM_SERVER
+          value: {{ .Values.serverCount | quote }}
+        - name: DMLC_PS_ROOT_URI
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        - name: DMLC_PS_ROOT_PORT
+          value: {{ .Values.schedulerPort | quote }}
+{{- if .Values.commonExtraEnvVars }}
+{{ include "mxnet.parseEnvVars" .Values.commonExtraEnvVars | indent 8 }}
+{{- end }}
+{{- if .Values.schedulerExtraEnvVars }}
+{{ include "mxnet.parseEnvVars" .Values.schedulerExtraVars | indent 8 }}
+{{- end }}
+        ports:
+        - name: mxnet
+          containerPort: {{ .Values.schedulerPort }}
+        {{- if .Values.livenessProbe.enabled }}
+        livenessProbe:
+          tcpSocket:
+            port: {{ .Values.schedulerPort }}
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.livenessProbe.successThreshold }}
+          failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+        readinessProbe:
+          tcpSocket:
+            port: {{ .Values.schedulerPort }}
+          initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.readinessProbe.successThreshold }}
+          failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
+        {{- end }}
+        resources: {{ toYaml .Values.resources | nindent 12 }}
```

Comment:
shouldn't you add the `{{ }}` code in the next line?

Reply:
Done

Reply:
ok